src/panels/lovelace/create-element/create-row-element.ts

```diff
@@ -15,6 +16,7 @@ const ALWAYS_LOADED_TYPES = new Set([
   "sensor-entity",
   "text-entity",
   "toggle-entity",
+  "weather-entity",
```

Comment:
I don't know if added this correctly to this file

Reply:
Make it lazy loaded